package.json

```diff
@@ -1,20 +1,37 @@
 {
-  "private": true,
-  "scripts": {
-    "prepare": "yarn tslint && lerna run prepare",
-    "rebuild:browser": "theia rebuild:browser",
-    "rebuild:electron": "theia rebuild:electron",
-    "tslint-fix": "tslint --fix --project github-extension",
-    "tslint": "tslint --project github-extension"
-  },
+  "name": "che-theia-github-plugin",
```

Comment:
maybe name should be @eclipse-che/theia-github-plugin

Reply:
done